src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment: { name, email, body } }) => (
+  <>
```

Comment:
what is the purpose of this fragment?

Reply:
To the component  CommentInfo we pass the object as a property "comment" from CommentList and destructorize it. Create the necessary markup for one element.